docs/metaprogramming/01_datatypes.md

```diff
@@ -205,14 +205,12 @@ Now, writing down `Expr` trees is very time consuming, because you have to manua
 For example,
 ```lean
 set_option pp.all true -- show all of the gory details.
-#check (λ x , x + 2 )
+#check ( λ x => x + 2 )
 /- outputs:
-λ (x : nat), @has_add.add.{0} nat nat.has_add x (@bit0.{0} nat nat.has_add (@has_one.one.{0} nat nat.has_one))
```

Comment:
thanks for updating this!

Reply:
I have now created a new pull request from the updated repository as you suggested.